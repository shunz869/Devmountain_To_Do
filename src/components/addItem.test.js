import React from 'react';
import AddItem from './addItem';
import renderer from 'react-test-renderer';

test('AddItem renders correctly', ()=>{
  const component = renderer.create(<AddItem item={{
    id:1,
    title:'abc'
  }} />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})