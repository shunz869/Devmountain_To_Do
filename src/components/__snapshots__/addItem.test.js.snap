// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddItem renders correctly 1`] = `
<form
  onSubmit={[Function]}
>
  <div
    className="form-group"
  >
    <input
      className="form-control"
      placeholder="please enter your title"
      type="text"
    />
  </div>
  <button
    className="btn btn-primary"
  >
    Add new To-do
  </button>
</form>
`;
